src/picongpu/include/plugins/particleMerging/ParticleMerger.kernel

```diff
@@ -0,0 +1,422 @@
+/* Copyright 2017 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "plugins/particleMerging/VoronoiCell.hpp"
+#include "particles/access/Cell2Particle.hpp"
+#include "memory/Array.hpp"
+#include "memory/IndexPool.hpp"
+
+namespace picongpu
+{
+namespace plugins
+{
+namespace particleMerging
+{
+
+using namespace PMacc;
```

Comment:
update: please don't do this, the namespace `PMacc` us used by default in PIConGPU